salmonella/static/salmonella/js/salmonella.js

```diff
@@ -1,15 +1,22 @@
 // Overwrite Django's `dismissRelatedLookupPopup` to trigger a change event
-// on the value change, so Salmonella can catch it and update the associated
-// label.
-function dismissRelatedLookupPopup(win, chosenId) {
+// on the value change, so Salmonella can catch it and update the associated label.
+// if an image is to be inserted call the .
+function dismissRelatedLookupPopup(win, chosenId, chosenObj) {
     var name = windowname_to_id(win.name);
     var elem = document.getElementById(name);
-    if (elem.className.indexOf('vManyToManyRawIdAdminField') != -1 && elem.value) {
+
+    if (name === 'id_image_embed' && chosenObj){
+      insertAssetIntoMDE(name, chosenObj);
```

Comment:
this assumes there is a `insertAssetIntoMDE` function, if this needs to be called, can we try to have something generic here and then call `insertAssetIntoMDE` from outside of this lib ?